modules/cpdos/hbh.py

```diff
@@ -106,7 +102,7 @@ def HBH(
                 else:
                     response_2_count_status_code += 1
 
-                # print(response_2_count_status_code)
+                logger.debug(response_2_count_status_code)
```

Comment:
Make the debug statements explicit via the logger use. These won't be seen by the users based on the configuration.